manifests/fedora-coreos-base.yaml

```
@@ -104,11 +104,10 @@ packages:
   # Containers
   - systemd-container catatonit
   - fuse-overlayfs slirp4netns
-  # support for old style CNI networks and name resolution for
-  # podman containers with CNI networks
+  # Some host applications((podman,NetworkManager, etc) ) use
```

The only known use now is NM

```suggestion
  # Some host applications(i.e. NetworkManager) use
```